features/Skylighting/Shaders/Skylighting/Skylighting.hlsli

```diff
@@ -7,15 +7,13 @@ struct SkylightingSettings
 	row_major float4x4 OcclusionViewProj;
 	float4 OcclusionDir;
 
-	float3 PosOffset;  // xyz: cell origin in camera model space
-	uint pad0;
-	uint3 ArrayOrigin;  // xyz: array origin, w: max accum frames
-	uint pad1;
+	float4 PosOffset;   // xyz: cell origin in camera model space
```

Comment:
Comment probably needs an update if float4.

Reply:
w is unused